File: Contest/Coder's Legacy 2017/Saving Up/main.py
test_cases = int(input())
for i in range(test_cases):
	no_of_expenditures,salary = map(int, input().split(" "))
	x = input()
	x = x.split(" ")
	sum_of_expenditures = 0
	for j in x:
		j = int(j)
		sum_of_expenditures += j
	if sum_of_expenditures <= salary/2:
		print("Yes")
	else:
		print("No")

File: Contest/Coder's Legacy 2017/Saving Up/main.c
#include <stdio.h>
#include <conio.h>

int main() {
	int test_cases,no_of_expenditures,salary,expenditure;
	scanf("%d",&test_cases);
	while(test_cases--) {
		int sum = 0;
		scanf("%d%d",&no_of_expenditures,&salary);
		while(no_of_expenditures--) {
			scanf("%d",expenditure);
			sum += expenditure;
		}
		printf("%d", sum);
	}
	return 0;
}